assets/blocks/learner-courses-block/learner-courses.scss

```diff
@@ -122,12 +122,6 @@ $courses-list: '#{$block}__courses-list';
 							margin: 0 7px;
 						}
 					}
-
-					#{$block} {
-						&__progress-bar {
-							height: 16px;
```

Comment:
Since now it's customizable by a setting, I removed the responsive height change.